src/geolocation/component.js

```diff
@@ -329,7 +330,7 @@ Controller.prototype.setPosition_ = function () {
 
   if (this.follow_) {
     this.viewChangedByMe_ = true;
-    if (this.options.zoom !== undefined) {
+    if (this.options.zoom) {
```

Comment:
```suggestion
    if (this.options.zoom or this.options.zoom === 0) {
```

Reply:
"or" is not a valid operator in javascript, it's not python. Plus zoom = 0 is not equal to what I want check, if the value is null.